src/backend/aspen/api/schemas/samples.py

```diff
@@ -60,7 +81,7 @@ class SampleGetterDict(GetterDict):
             if obj.uploaded_pathogen_genome
             else None
         ),
-        "lineage": format_sample_lineage,
+        "lineages": format_sample_lineage,
```

Comment:
Do we still need to use `format_sample_lineage` or can we rely on pydantic's built-in serialization now?

Reply:
it's a little nice for adding the `qc_score` to the lineages output  (to pull from the associated `sample.qc_metrics`) and also we still need to pull the scorpio calls from the `raw_lineage_output`, depending on sample.pathogen. So i think it's still worth having `format_sample_lineages`

Reply:
I think there might be a more pydantic-style way of serializing this object. Can you look into whether using `@validator()` methods can help us pull out the scorpio/other data we need? I think the `PhyloRunResponse` object has some interesting logic in it that might be a useful reference